src/AbstractFilter.php

```diff
@@ -21,6 +21,9 @@
 use function ucwords;
 
 /**
+ * @deprecated Since 2.38.0 - This class will be removed in version 3.0 without replacement. Custom filters should
+ *             implement FilterInterface without unnecessary inheritance.
```

Comment:
This information must also be added to the documentation: https://docs.laminas.dev/laminas-filter/v2/migration/preparing-for-v3/